src/sqlfluff/core/templaters/slicers/tracer.py

```diff
@@ -51,6 +52,7 @@ def __init__(
         raw_slice_info: Dict[RawFileSlice, RawSliceInfo],
         sliced_file: List[TemplatedFileSlice],
         render_func: Callable[[str], str],
+        config: Optional[FluffConfig] = None,
```

Comment:
Do we need to pass the whole config in here? I'm not necessarily against it, but it seems unnecessary at the moment. Are you doing something interesting with it in the derived case?

Reply:
Actually no.... not currently using it in the derived case (and don't currently expect to need to).  This was more of a future-proofing move.  I can remove it.

Reply:
Added on a commit that removes the config parameters, including from the _get_jinja_analyzer that I added earlier this year but don't really need it there either.